runner.rb

```diff
@@ -45,6 +45,23 @@ def show_entries
     puts "\nYou have logged #{@instahours.time_for_in_hours} hours so far for the week.\n\n"
   end
 
+  def get_date_from_array(entry_array)
+    begin
+      return Date.parse(entry_array[2])
```

Comment:
IF there is a date, it will be 3rd. If that is not a date, then return the current day